src/patternfly/components/AboutModalBox/about-modal-box.scss

```diff
@@ -188,6 +188,10 @@
   overscroll-behavior: contain;
   -webkit-overflow-scrolling: touch;
 
+  > * {
```

Comment:
the `>` is redundant since `word-break` is inherited.

Reply:
updated @mcoker